src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -516,7 +516,13 @@ object ProtocPlugin extends AutoPlugin {
                 f.getAbsolutePath()
               }
           }
-        s"--plugin=${dep.name}=${pluginPath}"
+        }
+        val targetCount = PB.targets.value.count {
+          case Target(PluginGenerator(name, _, _), _, _) => s"protoc-gen-$name" == dep.name
+          case _                                         => false
+        }
+        if (targetCount == 1) Seq(s"--plugin=${dep.name}=${pluginPath}")
```

Comment:
see discussion in https://github.com/scalapb/protoc-bridge/pull/160#discussion_r566327478